docs/api/rtos/Thread.md

```diff
@@ -10,6 +10,8 @@ The Thread class allows defining, creating and controlling parallel tasks.
 
 All the internal thread data structures are part of the C++ class, but by default, the thread stack is allocated on the heap. Memory is allocated at the run time during the call to `start` method. If you don't want to use dynamic memory, you can provide your own static memory using the constructor parameters.
 
+The main thread stack size is specified as `rtos.main-thread-stack-size` in the configuration .json file. That defines the main thread for `mbed_rtos_start` in `mbed_rtos_rtx.c`.
```

Comment:
Correct, but I think this needs to be the third paragraph. The paragraphs above and below are talking about extra threads you create using the `Thread` API, but this paragraph is a side point about the memory for the starting main thread (which is not created using `Thread`).

Possibly it should be a "Note", like in the section above.